packages/falcon-server-env/src/cache/Cache.ts

```diff
@@ -26,11 +26,14 @@ export type GetCacheOptions = {
   options?: SetCacheOptions;
 };
 
+// 1 hour
+const DEFAULT_TAG_TTL: number = 60 * 60;
```

Comment:
I am aware that it could be considered fussy but I would put comment from the line `29` into `30`
```suggestion
const DEFAULT_TAG_TTL: number = 60 * 60; // 1 hour
```

Reply:
sure, done